applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -150,7 +149,6 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
     int Check() override
     {
         KRATOS_TRY
-
         BaseType::Check();
         this->GetBuilderAndSolver()->Check(BaseType::GetModelPart());
         this->GetScheme()->Check(BaseType::GetModelPart());
```

Comment:
Do you know why we call the checks of the BaseType, which is ImplicitSolver and not of the direct base, which is the GeoMechanicsNewtonRaphsonStrategy (or the base of that which is ResidualBasedNewtonRaphsonStrategy)?

Reply:
Thank you for the question. I do not know. The difference between these Check functions is only one line, on convergence criteria. Calling the base class Check works OK. Definitely, it is worth to check during the refactoring.